openlibrary/plugins/upstream/utils.py

```diff
@@ -1311,6 +1312,24 @@ def get_location_and_publisher(loc_pub: str) -> tuple[list[str], list[str]]:
     return ([], [loc_pub.strip(STRIP_CHARS)])
 
 
+@cache.memoize(engine="memcache", key="edu_domains", expires=0)
+def get_edu_domains() -> list[str]:
+    """
+    This list created using this gist: https://gist.github.com/jimchamp/f67ef14c3bcf11593f393ee5288f6476 on
+    https://raw.githubusercontent.com/Hipo/university-domains-list/master/world_universities_and_domains.json.
+
+    See: https://github.com/internetarchive/openlibrary/issues/8180
+    """
+    p = Path(Path.cwd(), 'static', 'edu-domains.txt')
```

Comment:
(Side note; the `/` operator can be use here!)

```suggestion
    p = Path.cwd() / 'static' / 'edu-domains.txt'
```